### ISSUE TITLE
Need links to Understanding docs for guidelines

### ISSUE THREAD

**Issue by mmeller75** [w3c/wcag#595]
Links are needed to the Understanding documents for the guidelines from WCAG 2.1. 

Currently, it takes several steps to find those Understanding docs. For example, reaching "[Understanding Guideline 1.1: Text Alternatives](https://www.w3.org/WAI/WCAG21/Understanding/text-alternatives)" requires selecting the link "Understanding Non-text Content" under Success Criterion 1.1.1 Non-text Content, then choosing "GL: Text Alternatives" from the navigation at the top of the page.

**Comment by patrickhlauke** [comment]
Not quite sure from *where* you mean links are needed.

If, for instance, you're on https://www.w3.org/TR/WCAG21/#text-alternatives, there are links in the boxout right there to the understanding and "how to meet" documents, pointing directly to the correct place.

![Screenshot of WCAG 2.1 SC 1.1.1 showing the boxout with links to understanding and how-to-meet](https://user-images.githubusercontent.com/895831/51862733-286e4700-2337-11e9-9faa-2d92e1c9183d.PNG)

**Comment by patrickhlauke** [comment]
and there's a link to the main entry point for the understanding doc in the front matter section of the WCAG https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents

**Comment by mmeller75** [comment]
Yes, there is one to SC 1.1.1, but if you look above at the guideline, 1.1, there is no link to the understanding document.

![screen shot 2019-01-28 at 2 53 32 pm](https://user-images.githubusercontent.com/47121512/51862870-9271f680-230d-11e9-88d3-7121416f2a9c.png)

**Comment by patrickhlauke** [comment]
ah, now I get you. sorry, misread and thought you meant individual SCs. so yes, having some link from the guideline itself to https://www.w3.org/WAI/WCAG21/Understanding/text-alternatives ... makes sense

**Comment by mmeller75** [comment]
Those may not be used as much, and it is possible to get to them, but it takes a little work. I noticed in particular when I was looking for GL 1.2. A link right below the permalink, like with the SCs, would be very helpful.

**Comment by awkawk** [comment]
Unfortunately these are in the Recommendation so we may not be able to fix immediately. I assigned Michael to this as he will handle this in the generator.

**Comment by michael-n-cooper** [comment]
We can fix this when we do the Edited Recommendation, which will probably be soonish.

**Comment by awkawk** [comment]
Approved in survey and on call: https://www.w3.org/2019/02/12-ag-minutes.html#item03

**Comment by awkawk** [comment]
Approved in CfC Feb 21 2019. Leaving open for Michael to implement.

**Comment by fstrr** [comment]
@kfranqueiro assuming this can be closed.

**Comment by kfranqueiro** [comment]
> [@kfranqueiro](https://github.com/kfranqueiro) assuming this can be closed.

Yes, guidelines in the recommendation (both 2.1 and 2.2) now link to the understanding docs.
